### PR TITLE
use "container name witout project" for both logs and events

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**

adopted "container name without project" as the canonical name for both logs prefixes and lifecycle event

**Related issue**
https://github.com/docker/compose-cli/issues/1405

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**

### CODE REVIEW

File: aci/convert/volume.go [hunk 1]
```diff
@@ -139,7 +139,7 @@ type volumeInput struct {
 
 // parse takes a candidate string and creates a volumeInput
 // Candidates take the form of <source>[:<target>][:<permissions>]
-// Source is of the form `<storage account>/<fileshare>`
+// Container is of the form `<storage account>/<fileshare>`
```
Comment (gtardif) [inline aci/convert/volume.go:142]: a find/replace with too large scope ? (and I suppose many more instances following)

File: api/compose/api.go [hunk 2]
```diff
@@ -314,22 +314,21 @@ type Stack struct {
...
+	Log(service, container, message string)
+	Status(container, msg string)
+	Register(container string)
 }
 
 // ContainerEventListener is a callback to process ContainerEvent from services
 type ContainerEventListener func(event ContainerEvent)
 
-// ContainerEvent notify an event has been collected on Source container implementing Service
+// ContainerEvent notify an event has been collected on Container container implementing Service
```
Comment (gtardif) [inline api/compose/api.go:325]: same here

**Review by gtardif — APPROVED** [review]
Just fix unwanted replace of source/ Container. Otherwise LGTM

File: aci/convert/volume.go (hunk 1 above)
Comment (ndeloof) [reply aci/convert/volume.go:142]: ouch :'(

**Pull request merged**